### DISCUSSION TITLE
Searching for a copy of Trans mission Torent for Lenovo EMC NAS PX4-400D

### DISCUSSION THREAD

**Discussion by chiquito3** [transmission/transmission#6830] [Support]
Hello! I'm searching for a copy of the Transmission Torrent version for Lenovo NAS PX4-400D.
Could someone please help me?

**Comment by chiquito3** [comment]
Someone just told me that the Lenovo NAS I have OS is based on Devian 7.1.
Can someone suggest how to install the program manually?

**Reply by Pentaphon** [reply]
You may want to install a more modern OS like Ubuntu or Openmediavault which will make it easier to install Transmission and make your NAS more secure and compatible.

**Reply by chiquito3** [reply]
And how can I do that. This is a Lenovo NAS that has it's own OS. I do not think that it would be easy to install another OS that would work. 
If it is possible then I'm up to do it if someone tells me how to. The NAS has an Intel Atom D2701 dual-core processor.

**Reply by killemov** [reply]
If it is based on Debian, you can `apt install transmission-daemon` on the command line.

**Reply by chiquito3** [reply]
> If it is based on Debian, you can `apt install transmission-daemon` on the command line.

Hi killemov,

How can I do that? I do not know how.

**Reply by killemov** [reply]
NO! This is a YOU + device/Linux problem. So get educated on Linux, but not here. You at least need to search for ssh/shell access for your specific device. Good luck!
